Tasks/NpmAuthenticate/npmauth.ts

```diff
@@ -2,112 +2,104 @@ import * as path from 'path';
 import * as tl from 'vsts-task-lib/task';
 import * as URL from 'url';
 import * as fs from 'fs';
-import * as constants_1 from './constants';
-import * as npmregistry_1 from 'npm-common/npmregistry'
+import * as constants from './constants';
+import * as npmregistry from 'npm-common/npmregistry'
 import * as util from 'npm-common/util';
+import * as os from 'os';
+import * as npmrcparser from 'npm-common/npmrcparser';
 
 async function main(): Promise<void> {
     tl.setResourcePath(path.join(__dirname, 'task.json'));
-    let saveNpmrcPath;
-    let npmrc = tl.getInput(constants_1.NpmAuthenticateTaskInput.WorkingDir);
-    let workingDir = path.dirname(npmrc);
-    if (!(npmrc.endsWith('.npmrc') && tl.exist(npmrc))) {
+    let saveNpmrcPath: string;
+    let npmrc = tl.getInput(constants.NpmAuthenticateTaskInput.WorkingFile);
+    console.log(npmrc)
```

Comment:
Is this intentional? If so, add a message so that we know what that is when looking at build logs.